src/js/common/components/RequiredLoginView.jsx

```diff
@@ -65,23 +65,21 @@ class RequiredLoginView extends React.Component {
           return <SystemDownView messageLine1="We couldn’t find your records with that information." messageLine2="Please call the Vets.gov Help Desk at 1-855-574-7286. We’re open Monday‒Friday, 8:00 a.m.‒8:00 p.m. (ET)."/>;
         }
 
-        // If va_profile has any other value, continue on to check if this user can
-        // use this specific service.
-        if (this.isServiceAvailable()) {
-          // If you have the required service, show the application view.
-          return this.props.children;
-        }
-
+        // If va_profile has any other value, show the application view.
```

Comment:
This comment seems out of place now.

Am I reading the logic correctly that we go ahead and pass in `props = { isDataAvailable: this.isServiceAvailable() };` regardless of whether `this.isServiceAvailable()` evaluates to true or false to get the same effect as what's currently here?

I should be able to verify that, but it's late Friday afternoon so feeling lazy.

Reply:
Just looked at the specs and confirmed that yes, that's what's happening.

Reply:
Would you be willing to update the comments that refer to `va_profile` to make more sense? Otherwise, LGTM